weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallDetails.tsx

```diff
@@ -72,7 +75,136 @@ export const CallDetails: FC<{
             />
           </Box>
         )}
+        {multipleChildCallOpRefs.map(ref => (
+          <Box
+            key={ref}
+            sx={{
+              flex: '0 0 auto',
+              height: '500px',
+              maxHeight: '95%',
+              p: 2,
+              display: 'flex',
+              flexDirection: 'column',
+            }}>
+            <Box
+              sx={{
+                display: 'flex',
+                flexDirection: 'row',
+                flex: '0 0 auto',
+                alignItems: 'center',
+                justifyContent: 'space-between',
+                height: '50px',
+              }}>
+              <Typography sx={{display: 'flex', gap: 1}}>
+                Table of <SmallRef objRef={parseRef(ref)} wfTable="OpVersion" />{' '}
+                child calls
+              </Typography>
+              <IconButton
```

Comment:
Fine for now but we should probably be using our common component button and icons whenever we can.